src/sentry/conf/server.py

```diff
@@ -883,9 +883,9 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
         "options": {"expires": 3600},
     },
     "auto-enable-codecov": {
-        "task": "sentry.tasks.auto_enable_codecov.schedule_organizations",
-        # Run job every hour at min 20
-        "schedule": crontab(minute=20, hour="*/1"),
+        "task": "sentry.tasks.auto_enable_codecov",
```

Comment:
This may need to be:
```suggestion
        "task": "sentry.tasks.auto_enable_codecov.auto_enable_codecov",
```